tests/Fixer/Phpdoc/PhpdocLineSpanFixerTest.php

```diff
@@ -557,6 +557,151 @@ class Foo
         ];
     }
 
+    /**
+     * @dataProvider provideFix80Cases
+     * @requires PHP 8.0
+     */
+    public function testFix80(string $expected, string $input = null, array $config = []): void
```

Comment:
I think the reported error is for the `$config` parameter here.

Reply:
Okay, I guess I fixed it now then